ansible/artifacts/sunbird/login/info.ftl

```diff
@@ -12,9 +12,9 @@
                     <#if skipLink??>
                     <#else>
                         <#if pageRedirectUri??>
-                            <p><a href="${pageRedirectUri}">${msg("backToApplication")}</a></p>
+                            <p><a onclick="javascript:backToApplication()">${msg("backToApplication")}</a></p>
```

Comment:
We should not put any checks here. What ever the base or redirect uri is present. It should go there. Because this is generic page for any error accored by keycloak.